op-chain-ops/upgrades/check.go

```diff
@@ -65,7 +65,7 @@ func GetContractVersions(ctx context.Context, addresses *superchain.AddressList,
 	var versions superchain.ContractVersions
 	var err error
 
-	versions.L1CrossDomainMessenger, err = getVersion(ctx, common.HexToAddress(addresses.L1CrossDomainMessengerProxy.String()), backend)
```

Comment:
This is a drive by change which can be pinched off into a separate PR and then proliferated a bit more (there are many instances I didn't change yet).

Reply:
https://github.com/ethereum-optimism/optimism/pull/9599